### PR TITLE
a few edits for the docker console topic

### PR DESCRIPTION
**Pull request by mairaw**
I didn't have a chance to review the topic earlier today and I had a few suggestions, so created a PR to address them.

/cc @BillWagner @spboyer

### CODE REVIEW

File: docs/framework/docker/console.md [hunk 1]
```diff
@@ -86,7 +87,7 @@ ENTRYPOINT ConsoleRandomAnswerGenerator.exe
 The first line in the Dockerfile designates the base image using the [`FROM`](https://docs.docker.com/engine/reference/builder/#/from) instruction. Next, [`ADD`](https://docs.docker.com/engine/reference/builder/#/add) in the file copies the application assets from the **publish** folder to root folder of the container and last; setting the [`ENTRYPOINT`](https://docs.docker.com/engine/reference/builder/#/entrypoint) of the image states that this is the command or application that will run when the container starts. 
 
 ## Creating the image
-Adding the following code to the **build.ps1** script, when run the Docker image called `console-random-answer-generator` is created once the compilation is complete.
+Adding the following code to the *build.ps1* script, when run the Docker image called `console-random-answer-generator` is created once the compilation is complete.
```
Comment (mairaw) [inline docs/framework/docker/console.md]: @spboyer can you clarify what this sentence means? it's a bit confusing

**Comment by qinezh** [comment]
Open Publishing Build Service: The pull request content has been published and here are some changed files links of this time:
- Docset 1 - **core-docs**  
  [https://review.docs.microsoft.com/en-us/dotnet/articles/framework/docker/console?branch=pr-en-us-1100](https://review.docs.microsoft.com/en-us/dotnet/articles/framework/docker/console?branch=pr-en-us-1100)  

**Status**: SucceededWithWarning. **Message**: Warning occurred: Unable to resolve cross-reference "<a href="xref:Microsoft.Win32.SafeHandles.SafeNCryptKeyHandle" sourcefile="docs/standard/garbagecollection/implementing-dispose.md" sourcestartlinenumber="413" sourceendlinenumber="413">SafeNCryptKeyHandle</a>", "<a href="xref:Microsoft.Win32.SafeHandles.SafeNCryptProviderHandle" sourcefile="docs/standard/garbagecollection/implementing-dispose.md" sourcestartlinenumber="413" sourceendlinenumber="413">SafeNCryptProviderHandle</a>", "<a href="xref:Microsoft.Win32.SafeHandles.SafeNCryptSecretHandle" sourcefile="docs/standard/garbagecollection/implementing-dispose.md" sourcestartlinenumber="413" sourceendlinenumber="413">SafeNCryptSecretHandle</a>"..  
[Open Publishing Report](https://opbuildstorageprod.blob.core.windows.net/report/2016%5C10%5C1%5C2a032abb-a76c-bfc8-4231-71ab612989a2%5CPullRequest%5C201610010035084485-1100%5Cworkflow_report.html).

File: docs/framework/docker/console.md (hunk 1 above)
Comment (spboyer) [inline docs/framework/docker/console.md]: above when build.ps1 was introduced, it was to show how to use MSBuild to compile the application and produce the assets. This is the second portion of the script that is related to building the Docker image.

**Review by spboyer — APPROVED** [review]

**Comment by spboyer** [comment]
:shipit:

File: docs/framework/docker/console.md [hunk 2]
```diff
@@ -44,28 +44,29 @@ You can learn more about the Docker architecture by reading the
...
+
+![Windows-Containers](./media/console/SwitchContainer.png)
+
 ## Building the application
 Typically console applications are distributed through an installer, FTP, or File Share deployment. When deploying to a container, the assets need to be compiled and staged to a location that can be used when the Docker image is created.
 
-In **build.ps1** the script uses [MSBuild](https://msdn.microsoft.com/en-us/library/dd393574.aspx) to compile the application to complete the task of building the assets. There are a few parameters passed to MSBuild to finalize the needed assets. The name of the project file or solution to be compile,the location for the output and finally the configuration (Release or Debug).
+In *build.ps1*, the script uses [MSBuild](https://msdn.microsoft.com/en-us/library/dd393574.aspx) to compile the application to complete the task of building the assets. There are a few parameters passed to MSBuild to finalize the needed assets. The name of the project file or solution to be compile, the location for the output and finally the configuration (Release or Debug).
```
Comment (BillWagner) [inline docs/framework/docker/console.md]: Should this be "to be compiled" instead of "to be compile"?

**Review by BillWagner — CHANGES_REQUESTED** [review]

**Comment by BillWagner** [comment]
One typo remains, but everything else looks great.

File: docs/framework/docker/console.md (hunk 2 above)
Comment (mairaw) [inline docs/framework/docker/console.md]: I think so. I'll fix it.

File: docs/framework/docker/console.md (hunk 1 above)
Comment (mairaw) [inline docs/framework/docker/console.md]: Sure, but how can we improve the sentence? The flow is a bit weird after the comma.

**Comment by qinezh** [comment]
Open Publishing Build Service: The pull request content has been published and here are some changed files links of this time:
- Docset 1 - **core-docs**  
  [https://review.docs.microsoft.com/en-us/dotnet/articles/framework/docker/console?branch=pr-en-us-1100](https://review.docs.microsoft.com/en-us/dotnet/articles/framework/docker/console?branch=pr-en-us-1100)  

**Status**: SucceededWithWarning. **Message**: Warning occurred: Unable to resolve cross-reference "<a href="xref:Microsoft.Win32.SafeHandles.SafeNCryptKeyHandle" sourcefile="docs/standard/garbagecollection/implementing-dispose.md" sourcestartlinenumber="413" sourceendlinenumber="413">SafeNCryptKeyHandle</a>", "<a href="xref:Microsoft.Win32.SafeHandles.SafeNCryptProviderHandle" sourcefile="docs/standard/garbagecollection/implementing-dispose.md" sourcestartlinenumber="413" sourceendlinenumber="413">SafeNCryptProviderHandle</a>", "<a href="xref:Microsoft.Win32.SafeHandles.SafeNCryptSecretHandle" sourcefile="docs/standard/garbagecollection/implementing-dispose.md" sourcestartlinenumber="413" sourceendlinenumber="413">SafeNCryptSecretHandle</a>"..  
[Open Publishing Report](https://opbuildstorageprod.blob.core.windows.net/report/2016%5C10%5C4%5C2a032abb-a76c-bfc8-4231-71ab612989a2%5CPullRequest%5C201610040645007297-1100%5Cworkflow_report.html).

**Comment by mairaw** [comment]
@BillWagner does it look ok to you now? @spboyer please look at my last question.

File: docs/framework/docker/console.md (hunk 2 above)
Comment (BillWagner) [inline docs/framework/docker/console.md]: Still missing the 'd' on compiled:  "or solution to be compile, the location"

File: docs/framework/docker/console.md (hunk 1 above)
Comment (spboyer) [inline docs/framework/docker/console.md]: >Adding the following code to the *build.ps1* script, when run the Docker image called console-random-answer-generator` is created once the compilation is complete.

In order to create the Docker image, the following code is added to the *build.ps1* script. When run, the `console-random-answer-generator` image is created using the assets compiled from MSBuild previously defined in [Building the application](#building-the-application).

**Comment by spboyer** [comment]
@mairaw see inline comment

**Review by BillWagner — APPROVED** [review]

**Comment by BillWagner** [comment]
:shipit:

File: docs/framework/docker/console.md (hunk 1 above)
Comment (mairaw) [inline docs/framework/docker/console.md]: Much better @spboyer! I'll add this in a new PR. Thanks!

**Pull request merged**